scarb/src/core/config.rs

```diff
@@ -328,6 +338,11 @@ impl ConfigBuilder {
         self
     }
 
+    pub fn custom_source_patches(mut self, custom_self_patches: Vec<ManifestDependency>) -> Self {
```

Comment:
```suggestion
    pub fn set_custom_source_patches(mut self, custom_self_patches: Vec<ManifestDependency>) -> Self {
```

Reply:
actually all methods in this builder skip the `set_` part 😅